src/qc.rs

```diff
@@ -5,38 +5,40 @@
 
 use std::boxed::Box;
 use std::fmt;
+use core::{f32, f64};
 
 use quickcheck::{Arbitrary, Gen};
 
 use int::LargeInt;
+use float::Float;
 
 // Generates values in the full range of the integer type
 macro_rules! arbitrary {
-    ($TY:ident : $ty:ident) => {
+    ($id:ident: $ty:ty) => {
```

Comment:
I forgot that I left these changes in the commit. I can remove them if you prefer the current style.